tensorrt_llm/_torch/pyexecutor/self_benchmark.py

```
@@ -0,0 +1,2129 @@
+                    self._admission_records[trial_id]
+                    for trial_id in sorted(self._admission_records)
+                ],
+                "skipped_cases": [
+                    self._serialize_skipped_case(skipped_case)
+                    for skipped_case in self._skipped_cases
+                ],
+                "abort": self._serialize_abort(),
+                "results": [self._serialize_result(result) for result in self._results],
+            }
+            self._atomic_write_json(self._output_path, output)
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)